homework_part_2/javascript/app.js

```diff
@@ -0,0 +1,90 @@
+// Part 1
+// 1. Concatenation is the practice of combining multiple strings using the + operator.
+// Interpolation is a similar practice, but involves combining strings with variables that we've previously declared
+// 2. DRY stands for Don't Repeat Yourself. It's a best practice in programming, as it allows
+// for clean code that runs faster and avoids repetition throughout your code that takes time to write and takes longer to process
+// One tool that we've learned thus far is the practice of using loops, both for and while loops. This allows us to perform many iterations
+// of a block of code without having to write that code many times over.
+// 3. Declaring a variable does not necessarily involve assigning a value to it. To declare a variable num, we could simply say let num;
+// To assign value, we use the = operator i.e. let num = 2 or const num = 2.
+// To define a variable is to both declare and assign value to the variable.
+// 4. We should use const when we do not plan on redefining a variable, though this does not preclude us from mutating the variable
+// In most cases, we'll want to use const.
```

Comment:
yep -- use `const` until browser says "Uncaught TypeError: Assignment to constant variable." then change it

Reply:
and use `let` in a `for` loop obviously